tests/_support/Step/Acceptance/ShopSystem/GenericShopSystemStep.php

```diff
@@ -265,4 +276,23 @@ public function getActingPaymentMethod($paymentMethod): string
         }
         return $paymentMethod;
     }
+
+    public function logInToAdministrationPanel()
```

Comment:
Also noticed that phpdoc missing for this function

Reply:
Fixed in my last push.